governance/process.md

```diff
@@ -1,144 +1,209 @@
-    agrees to act as a "sponsor."
-      * Sponsor: takes responsibility to ensure that progress
-      is tracked and that outcomes are reported to the group, including
-      proposing to close the issue if there is not sufficient activity to
-      sustain the effort.
+      coordination may begin and set up time or expectations with others to
+      begin work.
+
+   Once the scope is defined and a definition of done is described, the
+   nominated lead(s) needs to seek out a Chair or Technical Lead to sponsor the
+   proposal. 
+   * The sponsor reviews the scope and definition of done, determines if it is
+     in scope of the roadmap, and updates the issue with the scope and
+     definition of done. 
+      * Sponsor: takes responsibility to ensure that progress is tracked and
+        that outcomes are reported to the group, including proposing to close
+        the issue if there is not sufficient activity to sustain the effort.
+    * The sponsor presents the proposal for discussion at the next leadership
+      meeting to determine roadmap concurrence and group workload.
       * Roadmap: determine there is interest and strong roadmap alignment,
-      but that there is not enough bandwidth to be confident that this work
-      can be driven to completion.
+        determine the workload of the group and existing bandwidth to ensure it
+        is driven to completion. If the proposal does not fit in the roadmap, it
+        does _not_ mean that it cannot be worked.
 
 1. **Accept or close the proposal.**
 
-   A) **Accept**: assign to the Sponsor and the Project Lead(s) working on the
-   effort, with members interested in contributing noted in the issue
-   descriptions, along with information about expected duration, milestones,
-   scope, and anticipated deliverables. An accepted
-   proposal becomes an active project (see below) and the "proposal" label
-   is removed, the "project" label is added and it is added to the backlog.
-
-   B) **Close**: a github comment on the issue should note the reason and
-    link to discussion minutes (when decision is reached at a group meeting)
-    or at least two members of the leadership team should be noted
-    in agreement (which may include the person who closes the issue).
-
-   C) **Roadmap**: the issue will remain a proposal and be placed on a
-   roadmap project board. The roadmap is reviewed quarterly.
+   A) **Accept**: The sponsor will confirm the project lead(s)and assign
+     themselves and the lead(s) to the issue, with members interested in
+     contributing noted in the issue descriptions, along with information about
+     expected duration, milestones, scope, and anticipated deliverables. An
+     accepted proposal becomes an active project (see below) and the "proposal"
+     label is removed, the "project" label is added and it is added to the
+     backlog.
+
+   B) **Close**: Closure may occur as a result of lack of interest (stale for 90
```

Comment:
When originally written, this option was considered way earlier in the process.  At any time, the proposal may be closed by leadership team as out of scope of the group, but we agreed not to do that before it was discussed at a meeting (hence original order) but it we put scoping first then this doesn't feel right here.

Reply:
A lot of suggestions or proposals that come to the group arent fully formed.  Scoping allows us to be more definitive in what is or is not aligned with the charter.

Reply:
yes, agreed for something that is clearly within our charter.  If something comes to us that is not appropriate to the group, then we want to close early in the process.